tutorials/commands/ssh-across-clusters.rst

```diff
@@ -0,0 +1,138 @@
+.. _ssh-across-clusters:
+
+===================
+SSH across clusters
+===================
+
+Let's say you want to create a Flux instance in an allocation on a cluster (e.g., let's say out first cluster is "noodle") 🍜️
+and then connect to it via ssh from another cluster (let's say our second cluster is called "quartz"). This is possible with the right
+setup of your ``~/.ssh/config``. 
+
+----------------------
+Create a Flux Instance
+----------------------
+
+First, let's create the allocation on the first cluster. We typically want to ask for an allocation,
+and run flux start via our job manager. Here we might be on a login node:
+
+.. code-block:: sh
+
+    # slurm specific
+    $ salloc -N4 --exclusive
+    $ srun -N4 -n4 --pty --mpibind=off flux start
+
+And then we get our allocation!
+As a sanity check, once you are on one of the nodes you should be able to submit a job and see the output:
+
+.. code-block:: sh
+
+    noodle:~$ flux mini run hostname
+    noodle220
+    noodle221
+    noodle222
+    noodle223
+
+And you might adopt this command to be more specific to your resource manager. E.g., slurm uses srun.
+After you run flux start, you are inside of a Flux instance! We generally want to launch 
+this instance as a parallel job under the native resource manager, rather than get an allocation and just run flux start.
+The reason is because the test instance ignores native resource allocation and starts N brokers in place.
```

Comment:
> The reason is because the test instance ignores native resource allocation and starts N brokers in place.

I think Jim was referring to when you were doing `flux start -s4` before.  In this case its not a test instance.  I think you can just remove this sentence.

Reply:
Is that not an interesting bit of knowledge to include? I thought it was.

Reply:
In an earlier iteration of your doc you used `flux start --test-size=4` (or maybe -s4)   And that's what Jims comment was about earlier when he said:

> Generally, you'd want to launch the instance as a parallel job under the native resource manager, rather than get an allocation and use flux-start --test-size=N. The test instance ignores the native resource allocation and just starts N brokers in place.

Now with `srun -N4 -n4 --pty --mpibind=off flux start`, this is a real flux instance against the native resource manager, not a test instance.   So the text seems out of place.

Reply:
okay, I'm good to remove then.